.github/workflows/deploy_branch_maven.reusable.workflow.yml

```diff
@@ -58,6 +58,11 @@ on:
         required: false
         type: string
         default: ''
+      harbor-registry-project-name:
```

Comment:
I order to keep consistent naming, input should be named 'docker-registry-project-name'

Reply:
ok, since this PR is already merged, I'll update directory in feature/v1 branch